examples/network_functions/vpp-ip-forwarder/vpp/values.yaml

```diff
@@ -0,0 +1,20 @@
+
+privileged: false
+
+image:
+  repository: soelvkaer/vppcontainer
+  tag: latest
+  pullPolicy: IfNotPresent
+
+volumeMounts:
+  vpp_sockets:
+    mountPath: /root/sockets
+
+cnf:
+  1:
+    main_core: 8
```

Comment:
We will need a real cluster to run this one.  I think Kind will die

Reply:
Yeah that could definitely be an issue. I will have to spin up a Kind cluster to test if this is the case, and if not specifying specific cores can remedy it.
Since this example is not intended for performance, having VPP dynamically assign cores should be sufficient (if that even works with Kind)

Reply:
@michaelspedersen, please add that to the README.

cc: @wavell